### ISSUE TITLE
Advanced units clarification

### ISSUE THREAD

**Issue by mr-ransel** [kawesomekhan/covid-indoor#37]
In the "Advanced" section, there is limited detail on the _meaning_ of the numbers provided. For example, "Ventilation System: 3" is a mystifying metric. Is that 3/10? 3 ACH? The third item in an arbitrary list somewhere? I found myself digging into the source code to find out how this number would be used, which seems unnecessary. The most intuitive (but not obvious) solution becomes that these fields are the same as the Basic units, but unlabeled: ACH, MERV, recirculation ACH. 

tl;dr: Add units to the Advanced Room Specification fields.

**Comment by kawesomekhan** [comment]
Yep - I just realized this slipped out when I started adding language translations a couple weeks ago. I added it back in just now.
